### PR TITLE
add validation

### PR DESCRIPTION
**Pull request by ruslan-teh**

### CODE REVIEW

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-add-service-pop-up/ubs-admin-tariffs-add-service-pop-up.component.html [hunk 1]
```diff
@@ -27,6 +30,9 @@ <h4>{{ 'ubs-tariffs-add-service.edit_service' | translate }}</h4>
           id="nameEng"
           placeholder="{{ 'ubs-tariffs-add-service.placeholder-service' | translate }}"
         />
+        <span class="validation" *ngIf="addServiceForm.controls['nameEng'].errors">{{
+          'ubs-tariffs-add-service.name-error' | translate
```
Comment (katia9261) [inline src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-add-service-pop-up/ubs-admin-tariffs-add-service-pop-up.component.html]: better to use here input-error-component

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-add-service-pop-up/ubs-admin-tariffs-add-service-pop-up.component.ts [hunk 2]
```diff
@@ -57,8 +57,8 @@ export class UbsAdminTariffsAddServicePopUpComponent implements OnInit, OnDestro
 
   createService() {
     return this.fb.group({
-      name: new FormControl('', [Validators.required, Validators.pattern(Patterns.NamePattern)]),
-      nameEng: new FormControl('', [Validators.required, Validators.pattern(Patterns.NamePattern)]),
+      name: new FormControl('', [Validators.required, Validators.pattern(Patterns.NamePattern), Validators.min(1), Validators.max(30)]),
```
Comment (katia9261) [inline src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-add-service-pop-up/ubs-admin-tariffs-add-service-pop-up.component.ts]: do we need here validation for a minimum value if this field is required?

Comment (ruslan-teh) [reply src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-add-service-pop-up/ubs-admin-tariffs-add-service-pop-up.component.ts]: i fixed  it

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-add-service-pop-up/ubs-admin-tariffs-add-service-pop-up.component.html (hunk 1 above)
Comment (ruslan-teh) [reply src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-add-service-pop-up/ubs-admin-tariffs-add-service-pop-up.component.html]: i fixed it

**Review by hnativlyubomyr — APPROVED** [review]

**Review by atsa21 — APPROVED** [review]

**Review by AlexW8719 — APPROVED** [review]

**Review by KKatiaa — APPROVED** [review]

**Review by katia9261 — APPROVED** [review]

**Review by NadiiaStoiko — APPROVED** [review]

**Pull request merged**